src/include/target/HGLRC_Hermes_2400_TX.h

```diff
@@ -1,8 +1,23 @@
 #define DEVICE_NAME "HGLRC Hermes 24"
 
+#define USE_SX1280_DCDC
```

Comment:
So we could add this to the E28 target now too then?

Reply:
Done!

And heres confirmation of the inductor. 
![image](https://user-images.githubusercontent.com/14170229/140641640-ba59fba8-4a01-4c90-b786-1beddf68cb14.png)